src/modules/Elsa.Studio.Workflows/Components/WorkflowInstanceViewer/Components/ActivityDetailsTab.razor.cs

```diff
@@ -24,17 +24,10 @@ public record ActivityExecutionRecordTableRow(int Number, ActivityExecutionRecor
     [Parameter] public JsonObject Activity { get; set; } = default!;
     
     /// The activity execution records.
```

Comment:
Let's update the description as well

Reply:
I don't follow. What updated description would do you gave in mind, and why?

Reply:
Do you mean that the object is activity execution **summary**?

Reply:
Updated to use the 'summary" wording. Let me know if you had something else in mind.

Reply:
I meant that the description above still refers to the collection of records: 
/// The activity execution records.

Instead of:
/// The last activity execution record.

Reply:
Ah, yes I get it now,  I was looking in the wrong file. Thanks.